backend/src/openarchiefbeheer/zaken/api/filtersets.py

```diff
@@ -49,10 +49,32 @@ class ZaakFilter(FilterSet):
         ),
     )
 
+    # Expand lookups
+
     zaaktype__omschrijving__icontains = CharFilter(
         field_name="_expand__zaaktype__omschrijving", lookup_expr="icontains"
     )
 
+    resultaat__resultaattype__omschrijving__icontains = CharFilter(
+        field_name="_expand__resultaat___expand__resultaattype__omschrijving",
+        lookup_expr="icontains",
+    )
+
+    resultaat__resultaattype__archiefactietermijn__icontains = CharFilter(
+        field_name="_expand__resultaat___expand__resultaattype__archiefactietermijn",
+        lookup_expr="icontains",
+    )
```

Comment:
For the exact filter, there was the 'bewaartermijn' filter. Maybe they can be unified :thinking: Also, I don't know how much sense it makes to have partial filtering for an ISO duration? :thinking:

Reply:
For filters I thing we should either 

- Have a partial filter
- Have a dropdown to select the exact value

I'm not sue how to unify this with the exact filter btw.